core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/unixtime/UnixTimeUtilsUnitTest.java

```diff
@@ -0,0 +1,106 @@
+package com.baeldung.unixtime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoField;
+import java.util.Calendar;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+public class UnixTimeUtilsUnitTest {
```

Comment:
Since we are using exclusively JUnit 5, there's no need now to have this class defined as public, so you can remove the modifier.